src/Microsoft.Identity.Client/Internal/Requests/AuthenticationRequestParameters.cs

```diff
@@ -130,13 +132,32 @@ public void LogState()
 #if DESKTOP || NETSTANDARD1_3
             builder.AppendLine("Confidential Client? - " + (ClientCredential != null));
             builder.AppendLine("Client Credential Request? - " + IsClientCredentialRequest);
-            if(IsClientCredentialRequest)
+            if (IsClientCredentialRequest)
             {
                 builder.AppendLine("Client Certificate Provided? - " + (ClientCredential.Certificate != null));
             }
 #endif
-            RequestContext.Logger.Info(builder.ToString());
+            RequestContext.Logger.InfoPii(builder.ToString());
 
+            // Create no Pii enabled string builder
+            builder = new StringBuilder(Environment.NewLine + "=== Request Data ===" +
+                                                      Environment.NewLine);
+            builder.AppendLine("Scopes - " + Scope?.AsSingleString());
+            builder.AppendLine("Validate Authority? - " + ValidateAuthority);
+            builder.AppendLine("LoginHint provided? - " + !string.IsNullOrEmpty(LoginHint));
+            dict = MsalHelpers.ParseKeyValueList(ExtraQueryParameters, '&', true, RequestContext);
+            builder.AppendLine("Extra Query Params Keys (space separated) - " + dict.Keys.AsSingleString());
+            dict = MsalHelpers.ParseKeyValueList(ExtraQueryParameters, '&', true, RequestContext);
+            builder.AppendLine("Slice Parameters Keys(space separated) - " + dict.Keys.AsSingleString());
```

Comment:
same data are logged as "Extra Query Params Keys"

Reply:
removed for pii.